third-party/lzokay/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.10)
+include(CMakePackageConfigHelpers)
+
+project(lzokay VERSION 0.1 LANGUAGES CXX)
+add_library(lzokay lzokay.hpp lzokay.cpp)
```

Comment:
I think we should make this `add_library(lzokay SHARED lzokay.hpp lazkay.cpp)` and it will fix the weird build issue.